templates/sub/ci_gitlab/{{cookiecutter.__folder_name }}/.gitlab-ci.yml

```diff
@@ -0,0 +1,175 @@
+default:
+  image: docker:24.0.5
+  services:
+    - docker:24.0.5-dind
+  before_script:
+    - apk add --no-cache make bash python3 ncurses openssh-client-common
+    - echo "$CI_REGISTRY_PASSWORD" | docker login $CI_REGISTRY -u $CI_REGISTRY_USER --password-stdin
+
+stages:
+  - check
+  - test
+  - build
+  - deploy
+
+variables:
+  # Use TLS https://docs.gitlab.com/ee/ci/docker/using_docker_build.html#tls-enabled
+  # DOCKER_HOST: tcp://docker:2376
+  DOCKER_TLS_CERTDIR: "/certs"
+  CONTAINER_BACKEND_RELEASE_IMAGE: $CI_REGISTRY_IMAGE/backend
+  CONTAINER_FRONTEND_RELEASE_IMAGE: $CI_REGISTRY_IMAGE/frontend
+  {%- if cookiecutter.devops_cache == '1' %}
+  CONTAINER_VARNISH_RELEASE_IMAGE: $CI_REGISTRY_IMAGE/varnish
+  {%- endif %}
+
+check-backend:
+  stage: check
+  before_script:
+    - apk add --no-cache make bash python3 ncurses openssh-client-common git
+  script:
+    - cd backend
+    - make check
```

Comment:
Where is the check-target in the Makefile? Do you mean `make lint`?

Reply:
yes, I meant make lint. I had an older version of cookieplone-templates in place where `make check` was the command, sorry about that :(